layers/meta-balena-coral/recipes-kernel/linux/linux-imx_4.14.98.bb

```diff
@@ -17,11 +17,11 @@ COMPATIBLE_MACHINE = "(mx6|mx7|mx8)"
 
 SUMMARY_coral-dev = "Linux Kernel provided and supported by Google for Coral Dev Board"
 
-SRCBRANCH_coral-dev = "release-day"
-SRCREV_coral-dev = "f19426310e2e0ff490a3160a64a05018d7f6c536"
+SRCBRANCH_coral-dev = "master"
+SRCREV_coral-dev = "c13f50afd3920f718f9becc9f47c31fccba00bf7"
 
 SRC_URI_coral-dev = "\
-    git://coral.googlesource.com/linux-imx;protocol=https;branch=${SRCBRANCH} \
+    git://coral.googlesource.com/linux-imx;protocol=https;branch=${SRCBRANCH_coral-dev} \
```

Comment:
You don't need this change, as the suffix after the underscore is only for assignment. Anywhere the variable is used will be correct for the machine name specified by the assignment.

Reply:
Hey, @klutchell. Thanks for checking that. That's indeed something I don't fully understand and I only added because there's also a `SRCBRANCH_asus-tinker-edge-t` and in line 36 it's using that with suffix, so I thought it would make sense to add that especially when there's two machines in the same file.

Can you please take a look at the full file for context? Check line 36. 
Please check if the change I made is indeed undesired or if it makes the file look more uniform. 
Maybe it's the line 36 the "wrong" one and mislead me.